src/v2/Apps/__tests__/Fixtures/Order.ts

```diff
@@ -66,7 +66,7 @@ const OrderArtworkNodeWithoutShipping = {
   is_offerable: true as boolean,
   listPrice: {
     __typename: "Money",
-    major: 10000,
+    major: 16000,
```

Comment:
I think we didn't use this before, but now that we are, it was inconsistent with the total price marked on the file.